homework-submission/week1/style.css

```diff
@@ -0,0 +1,26 @@
+.main {
+  background-color: rgb(144, 211, 233); 
+  font-size: 18px;  
+}
+
+.static_list {
+  background-color: rgb(17, 97, 124);
+  color: white;
+  margin: 10px;
+  padding: 10px;
+}
+
+.dynamic_list {
+  background-color: rgb(17, 97, 124);
+  color :white;
+  margin: 10px;
+  padding:10px;
+}
+
+.done {
+  text-decoration: underline;
```

Comment:
`line-through` is more clear that a todo item is done

Reply:
Certainly..